code/__DEFINES/DNA.dm

```diff
@@ -179,3 +179,6 @@
 #define G_MALE 1
 #define G_FEMALE 2
 #define G_PLURAL 3
+
+//Size / height stuff
+#define SPECIES_HEIGHTS(x, y, z) list("Normal" = x, "Short" = y, "Tall" = z)
```

Comment:
The ordering of normal, short, tall doesn't make much sense. Short, normal, tall would be more sensible.